features/steps/ubuntu_advantage_tools.py

```diff
@@ -312,3 +312,111 @@ def when_i_check_apt_cache_policy(context):
     for step in context.scenario.steps:
         if step.name == APT_POLICY_IS:
             step.text = context.process.stdout
+
+
+@when("I install transition package ubuntu-advantage-tools")
+def when_i_install_transition_uat(context, machine_name=SUT):
```

Comment:
This is mostly a copy-past of `when_i_install_uat`, so I think we can refactor this to reuse some code.

Can you turn most of this into a step called "When I set up the source for ubuntu-advantage-tools" which would only add PPAs or set up -proposed as necessary (and not install anything yet). Then the next step in this new test would be "When I apt install `ubuntu-advantage-tools`". We could also reuse that function inside of `when_i_install_uat`

Reply:
forgot to tag: @lucasmoura

Reply:
That's a great idea. I have updated the code